app/views/countries/partials/_habitats-present.html.erb

```diff
@@ -2,17 +2,25 @@
   <div class="flex">
     <h2><%= t('countries.shared.habitats_present.title'); %></h2>
   </div>
-  <p><%= t('countries.shared.habitats_present.intro_text', country_name: t("countries.#{@yml_key}.title")); %></p>
+  <p><%= t('countries.shared.habitats_present.intro_text', country_name: @name); %></p>
 
   <div class="cards--habitats-present">
     <% habitats_present.each do |habitat| %>
+      <!-- Present but data deficient habitats are essentially considered as 'Present' -->
+      <% if habitat[:status] = 'present-but-unknown'  %>
+        <% habitat[:status] = 'present' %>
+        <% habitat[:status_title] = 'Confirmed Presence' %>
+      <% end %>
```

Comment:
Can this be incorporated into a helper or similar? Perhaps in the HabitatsPresentSerializer (if that's where this is coming from)